linux/metainfo.xml

```diff
@@ -25,22 +25,36 @@ SPDX-License-Identifier: CC-BY-4.0
     <content_attribute id="violence-cartoon">mild</content_attribute>
     <content_attribute id="violence-fantasy">mild</content_attribute>
   </content_rating>
-  <url type="homepage">https://endlessaccess.org</url>
+  <url type="homepage">https://access.endlessstudios.com/threadbare</url>
   <url type="bugtracker">https://github.com/endlessm/threadbare/issues</url>
+  <url type="contact">https://github.com/endlessm/threadbare/discussions/categories/q-a</url>
+  <url type="vcs-browser">https://github.com/endlessm/threadbare</url>
+  <url type="contribute">https://github.com/endlessm/threadbare/contribute</url>
   <screenshots>
     <screenshot type="default">
-      <image>https://raw.githubusercontent.com/endlessm/threadbare/8ce8a7333aab9add931c02f9f23c81058acbb243/docs/screenshot.png</image>
+      <image>https://media.githubusercontent.com/media/endlessm/threadbare/67b2cad522c26b56abbfa334c2553a87d9b54af0/docs/screenshot.png</image>
     </screenshot>
   </screenshots>
   <releases>
-    <release version="0.0.0" date="2025-04-28">
-      <description>
-        <p>Pre-release</p>
-      </description>
+    <release version="0.1.3" date="2025-08-11">
+      <url type="details">https://github.com/endlessm/threadbare/releases/tag/v0.1.3</url>
+    </release>
+    <release version="0.1.2" date="2025-07-07">
+      <url type="details">https://github.com/endlessm/threadbare/releases/tag/v0.1.2</url>
+    </release>
+    <release version="0.1.1" date="2025-06-13">
+      <url type="details">https://github.com/endlessm/threadbare/releases/tag/v0.1.1</url>
+    </release>
+    <release version="0.1" date="2025-05-23">
+      <url type="details">https://github.com/endlessm/threadbare/releases/tag/v0.1</url>
     </release>
   </releases>
   <recommends>
     <control>keyboard</control>
     <control>gamepad</control>
+    <display_length side="shortest" compare="ge">1080</display_length>
```

Comment:
Oh, actually, since:

- https://github.com/endlessm/threadbare/pull/1009

We do actually have a required minimum size. I'll refresh this to add that.